code/graphics/paths/NanoVGRenderer.cpp

```diff
@@ -21,6 +21,22 @@
 #include "NanoVGRenderer.h"
 #include "tracing/tracing.h"
 
+// That is a wrapper function for log prints to be availiable for nanovg components. For now it is stb_truetype.h
+// Planted by ksotar with blessing from asarium
+extern "C" {
+void nvgOldCPrintf(SCP_FORMAT_STRING const char *message, ...) {
+	SCP_string buf;
+	va_list args;
+	va_start(args, message);
+	vsprintf(buf, message, args);
+	va_end(args);
+	#ifdef outwnd_printf2
```

Comment:
I'm assuming you are using this to check if we are in debug mode. In FSO this is usually done with `#ifndef NDEBUG`. Also, you can put the entire body of this function in that preprocessor block to save some memory in release mode.